bin/render-activity-stream-html.js

```diff
@@ -97,6 +97,7 @@ function main() {
 
   const options = Object.assign({}, DEFAULT_OPTIONS, args || {});
   const {html, state} = prerender(options.locale);
+  options.direction = state.App.textDirection;
```

Comment:
maybe remove the default / command line option if this is generated from the locale to prevent confusion?

Reply:
Removed `direction` from `minimist` and `DEFAULT_OPTIONS`